src/components/Tag/Tag.js

```diff
@@ -52,13 +56,18 @@ const tagClickableStyles = ({ onClick, theme }) =>
 
     &:hover {
       background-color: ${theme.colors.n300};
+      ${shadowBorder(theme.colors.n300)};
     }
   `;
 
 const iconStyles = ({ theme }) => css`
   label: tag__icon;
   vertical-align: -2px;
```

Comment:
I'm not a big fan of magic numbers here. Can't we make it a variable then reuse it on the `svg inherit` declaration after it? (This way we can avoid the inherit at all)

Reply:
Is this necessary because of the `display: inline-block`?